src/common/commonutils/DeviceInfoUtils.c

```diff
@@ -947,4 +947,22 @@ bool IsCommodore(void* log)
     FREE_MEMORY(textResult);
 
     return status;
-}
\ No newline at end of file
+}
+
+enum SelinuxState {
+  SelinuxUnknown = 0,
+  SelinuxFound,
+  SelinuxNotFound,
+};
+static enum SelinuxState g_selinuxState = SelinuxUnknown;

```

Comment:
The SelinuxState can be simplified to a Boolean if we'll initialize the global var to false aka no SELinux.

Reply:
switched it to a boolean, but there is a caveat - see comment below.